app/src/main/res/layout/item_bill_customer.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:bind="http://schemas.android.com/apk/res-auto"
+    >
+
+    <data>
+
+        <variable
+            name="bill"
+            type="com.framgia.fsalon.data.model.BillResponse"/>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.fsalon.screen.billcustomer.BillCustomerViewModel"
+            />
+    </data>
+
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/dp_2"
+        app:cardBackgroundColor="@color/color_white"
+        app:cardElevation="@dimen/dp_5">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginBottom="@dimen/dp_10"
+            android:layout_marginEnd="@dimen/dp_10"
+            android:layout_marginLeft="@dimen/dp_10"
+            android:layout_marginRight="@dimen/dp_10"
+            android:layout_marginStart="@dimen/dp_10"
+            android:layout_marginTop="@dimen/dp_10"
+            android:onClick="@{() -> viewModel.onBillDetailClick(bill)}"
+            android:orientation="horizontal">
+
+            <LinearLayout
+                android:layout_width="wrap_content"
```

Comment:
WARNING: [InefficientWeight] Use a `layout_width` of `0dp` instead of `wrap_content` for better performance
 When only a single widget in a LinearLayout defines a weight, it is more efficient to assign a width/height of `0dp` to it since it will absorb all the remaining space anyway. With a declared width/height of `0dp` it does not have to measure its own size first.